routes/websocket.js

```diff
@@ -157,6 +157,41 @@ export default function init(server, wss) {
 			return;
 		}
 
+		m = request.nc_url.pathname.match(/^\/ws\/room\/admin\/([a-zA-Z0-9-]+)/);
+
+		request.is_secret_admin = !!m;
+
+		if (request.is_secret_admin) {
```

Comment:
This should probably only enabled on local servers. Seems wrong to allow admin control by secret in production.